setup.cfg

```diff
@@ -8,7 +8,10 @@ copyright-regexp = # Copyright \(c\) Microsoft Corporation and Fairlearn contrib
 # An update to pip has resulted in `pip install .` leaving a 'build' directory around
 # Also: 'pip freeze' does not report the pip version by default
 exclude =
+    .git/
     build/
+    docs/
```

Comment:
We probably should keep `flake8` running on `docs/conf.py`.

Reply:
That is true, I replaced `docs/` with `docs/auto_examples` instead. Would this work?

Reply:
That should do it!